src/index.ts

```diff
@@ -317,12 +329,13 @@ export class Probot {
             url: this.options.webhookProxy,
           });
         }
-        this.log.info("Listening on http://localhost:" + this.options.port);
+        this.log.info(`Listening on http://${printableHost}:${port}`);
       })
       .on("error", (error: NodeJS.ErrnoException) => {
         if (error.code === "EADDRINUSE") {
           this.log.error(
-            `Port ${this.options.port} is already in use. You can define the PORT environment variable to use a different port.`
+            `Host:port ${printableHost}:${port} is already in use. ` +
+              `You can define the HOST and PORT environment variables to use a different host and/or port.`
```

Comment:
I'm not sure if changing `HOST` would help if you run a process on the same machine and the `PORT` is already in use. I think we can leave the old message here. Or am I missing something?

Reply:
A host (think "server" more than "workstation") could have multiple IP addresses configured (even on the same network interface) and each IP address basically creates its "TCP ports" namespace, e.g. one process could bind to `10.0.0.1:3000` and another to `10.0.0.2:3000`. Sure it's not a very common setup but afaik it's a possible one.

Reply:
I see. Still, in this case, if you have such a complex setup, you know what you are doing. I'd prefer not to confuse the users in the most common case, where the problem can only be addressed by configuring `PORT`